portfolio-project-matching-app/components/InputDropdown.js

```diff
@@ -0,0 +1,13 @@
+const InputDropdown = ({ choices, name, onChange }) => {
```

Comment:
There is a ton of aesthetic work that can be done to this component. There is a `react-select` package designed specifically for styling these dropdowns. At the moment, I've left things rather similar to the text input created by Charlie.